src/steps/10-FinancialDetails/IGCE/TravelEstimates.vue

```diff
@@ -1,15 +1,129 @@
 <template>
-  <div>
-    Future page for Travel Estimates
-  </div>
+  <v-container fluid class="container-max-width">
+    <v-row>
+      <v-col class="col-12">
+        <h1 class="page-header">
+          How do you want to estimate your travel needs?
+        </h1>
+        <p class="page-intro">
+          Consider the travel requirements that you previously outlined (e.g.,
+          location, duration, quantity, attendance, etc.). You may choose to set
+          a ceiling price and apply it to all periods. Or, if you know that your
+          travel requirements will change over time, then you may opt to set a
+          different price for each period.
+        </p>
+        <div class="copy-max-width">
+          <ATATRadioGroup
+            class="copy-max-width mb-10 max-width-740"
+            id="TravelEstimates"
+            :card="true"
+            :items="travelEstimateOptions"
+            @click="travelFormFields"
+            :value.sync="selectedTravelEstimate"
+            :rules="[$validators.required('Please select an option')]"
+          />
+        </div>
+
+      
+         <fieldset class="no-border" v-if="travelFormFields !==''">
+            <legend class="mb-4"><strong>Estimated travel costs per period</strong></legend>
```

Comment:
I should have noticed this before too, sorry... when single input, the label "Estimated travel costs..." is normal weight. When entering for each period, it goes to `font-weight-500` not `<strong>` which is 700
<img width="389" alt="Screen Shot 2022-10-06 at 10 41 24 AM" src="https://user-images.githubusercontent.com/90333349/194343031-6d7eeb97-6dac-4b4b-a675-3cd501f51ae5.png">